src/api/catalogueCategory.tsx

```diff
@@ -45,18 +45,15 @@ export const useCatalogueCategories = (
   isLeaf: boolean,
   parent_id: string
 ): UseQueryResult<CatalogueCategory[], AxiosError> => {
-  return useQuery<CatalogueCategory[], AxiosError>(
-    ['CatalogueCategories', parent_id],
-    (params) => {
+  return useQuery({
+    queryKey: ['CatalogueCategories', parent_id],
+
```

Comment:
```suggestion
```
I'm not keen on the spacing, mostly because it is a bit inconsistent in the other files below.

Reply:
I can remove that line but it is the same in all of the relevant ones changed. I can't see the inconsistency

Reply:
useDeleteItem doesn't have spaces at all, most onError/onSuccess are grouped together without spaces, but query keys, fns & enabled have spaces between them, I would personally prefer all of them to just be next to each other as they are properties and are all quite short

Reply:
Thanks for the clarification